src/components/queryEditor.tsx

```diff
@@ -471,6 +473,29 @@ export function QueryEditor(props: EditorProps) {
         {tab === Tabs.Timeseries && <TimeseriesTab {...{ onQueryChange, query, datasource }} />}
         {tab === Tabs.Custom && <CustomTab {...{ onQueryChange, query, onRunQuery }} />}
         {tab === Tabs.Event && <EventsTab {...{ onQueryChange, query, onRunQuery }} />}
+        {tab === Tabs.Relationships && (
+          <Relationships
+            {...{
+              query,
+              datasource,
+              onQueryChange,
+              className: 'full-width-row',
+              selectors: [
+                {
+                  rout: 'relationsShipsQuery.dataSetIds',
+                  type: 'datasets',
+                  keyPropName: 'id',
+                },
+                {
+                  rout: 'relationsShipsQuery.labels.containsAny',
+                  type: 'labels',
+                  keyPropName: 'externalId',
+                },
```

Comment:
move "selectors" inside the `Relationships` component.

Reply:
The selector field is passed, because the component was/will be reused in the TimeseriesTab for the relationships support with their own selector properties, I could adjust the selctors in the constanst and been "reselcetd" with a  "coponentType" variable to we can reuse the component with own selector properties. How do you see it?

Reply:
ok, I see your point now